python/knowledges/batch_upload/batch_upload_advanced.py

```
@@ -0,0 +1,659 @@
+        
+        if total_files == 0:
+            self.logger.info("No files to upload")
+            return
+        
+        self.tasks_queue = deque(all_tasks)
+        # 取得已完成的檔案數量（如果有 checkpoint）
+        if checkpoint:
+            completed_files = set(checkpoint.get('completed_files', []))
+            initial_completed = len(completed_files)
+        else:
+            initial_completed = 0
```

Ruff (0.12.2)

Local variable `initial_completed` is assigned to but never used

Remove assignment to unused variable `initial_completed`

(F841)